packages/page-accounts/src/Accounts/BannerExtension.tsx

```diff
@@ -53,10 +53,23 @@ function BannerExtension (): React.ReactElement | null {
   return (
     <Banner type='warning'>
       <p>{t<string>('One or more extensions are detected in your browser, however no accounts have been injected.')}</p>
-      <p>{t<string>('Ensure that the extension has accounts, some accounts are visible globally and available for this chain and that you gave the application permission to access accounts from the extension to use them.')}</p>
-      <p>{phishing.current}</p>
+      <p>
+        {t<string>('Ensure that:')}
+        <SafetyInfoList>
+          <li>{t<string>('the extension has accounts,')}</li>
+          <li>{t<string>('at least one account is available for this chain,')}</li>
+          <li>{t<string>('the extension allows azero.dev to access accounts')}</li>
+        </SafetyInfoList>
+      </p>
+      <p>{t<string>('For extra protection, consider using the Threat Slayer extension which protects you from dangerous websites in real-time.')}</p>
```

Comment:
Currently we only show this message if user has an extension and no accounts are injected, which is basically almost never. Perhaps we'd want to show this more frequently?

Reply:
I'll raise the question on slack.

Reply:
Updated after a discussion with marketing:
![image](https://github.com/Cardinal-Cryptography/azero-dev/assets/6209244/0f6459a3-c258-4519-8d8d-f353cbd9cafe)
![image](https://github.com/Cardinal-Cryptography/azero-dev/assets/6209244/aac78072-00af-4449-bb68-48d1995da2b1)

Reply:
Just noticed I had the one paragraph duplicated - fixed already.